data-generation/src/main/scala/utils/Benchmark.scala

```diff
@@ -7,21 +7,23 @@ import scala.concurrent.duration.NANOSECONDS
 /** Benchmark
  * Can be used like this:
  * {{{
- *   val b = Benchmark(ConsoleWriter, textPrefix="doStuff")
+ *   val b = Benchmark(ConsoleWriter, textPrefix="doStuff", customColumn="number of something")
  *   b.writeHeader
- *   b.benchmarkSingle({spark.doStuff})
+ *   b.benchmarkSingle({spark.doStuff}, customColum={someCalculation})
  *   b.textPrefix = "SomeOtherFunction"
- *   b.benchmarkAvg({spark.someOtherFunction}, 10)
- *   b.benchmarkAvg({spark.lastFunction},5, "lastFunction")
+ *   b.benchmarkAvg({spark.someOtherFunction}, 10, customColum={someCalculation}))
+ *   b.benchmarkAvg({spark.lastFunction},5, "lastFunction", customColum={someCalculation}))
  *   b.close() // should be called if its a FileWriter
  * }}}
  * textPrefix is a mutable variable that sets the name for the output format
  *
- * @param writer    IoC, write to file or to console
- * @param inSeconds if seconds is the preferred output unit of time (Could actually just be removed)
+ * @param writer       IoC, write to file or to console
+ * @param inSeconds    if seconds is the preferred output unit of time
+ * @param textPrefix   The function that is run
+ * @param customColumn A column where the benchmark user decides the content
  */
-case class Benchmark(writer: Writer, inSeconds: Boolean = false, textPrefix: String) {
-  val header = "function,value,timeunit,number of runs"
+case class Benchmark(writer: Writer, inSeconds: Boolean = false, textPrefix: String, customColumn: String = "") {
```

Comment:
Støttes! Hvis man vil være skikkelig kul gjør an det om til en `customColumns:Seq[String]` sånn at man kan legge til enda flere kolloner. Men er kanskje ikke så mye bruk for det